src/main/java/se/knowit/bookitregistration/controller/RegistrationController.java

```diff
@@ -42,23 +42,29 @@ public RegistrationController(RegistrationService service) {
                 .map(mapper::toDTO)
                 .collect(toSet());
     }
-    
+
     @PostMapping(value = {"", "/"}, consumes = "application/json", produces = "application/json")
     public ResponseEntity<String> registerToEvent(@RequestBody RegistrationDTO dto) {
         RegistrationSaveResult result = trySave(dto);
         return buildResponseFromRegistrationSaveResult(result);
     }
-    
+
     @DeleteMapping("/{id}")
-    public ResponseEntity<String> delete(@PathVariable String id) {
+    public ResponseEntity<String> deleteByRegistrationId(@PathVariable String id) {
         service.delete(id);
         return ResponseEntity.noContent().build();
     }
-    
+
+    @DeleteMapping("/{email}/{eventId}")
```

Comment:
I think that no request will ever go to this endpoint since they will be handled by the `@DeleteMapping("/{id}")` since / can be a part of the path variable `ìd`. 

I think if using multiple input parameters one could send these in the body. What are your thoughts?

Reply:
Tests have shown that this method works